README.md

```diff
@@ -1,44 +1,244 @@
+
+```
+(cob_datapipeline) $ airflow initdb
+  [2019-03-21 14:34:26,312] {__init__.py:51} INFO - Using executor SequentialExecutor
+  DB: sqlite:////home/tul08567/airflow/airflow.db
+  [2019-03-21 14:34:26,503] {db.py:338} INFO - Creating tables
+  INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
+  ...
+  Done.
+```
+
+Finally, start running the airflow webserver and the airflow scheduler (you'll need multiple shell sessions during local development, or to run these as background jobs):
+
+```
+(cob_datapipeline) $ airflow webserver -p 8080
+  [2019-03-21 14:44:47,863] {__init__.py:51} INFO - Using executor SequentialExecutor
+    ____________       _____________
+   ____    |__( )_________  __/__  /________      __
+  ____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
+  ___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
+   _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
+
+  [2019-03-21 14:44:48,130] {models.py:273} INFO - Filling up the DagBag from /home/tul08567/airflow/dags
+  Running the Gunicorn Server with:
+  Workers: 4 sync
+  Host: 0.0.0.0:8080
+  Timeout: 120
+  Logfiles: - -
+  ... # running process in this shell
+# In a new shell, in Pipenv:
+$ pipenv shell
+(cob_datapipeline) $ airflow scheduler
+  [2019-03-21 14:46:44,609] {__init__.py:51} INFO - Using executor SequentialExecutor
+    ____________       _____________
+   ____    |__( )_________  __/__  /________      __
+  ____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
+  ___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
+   _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
+
+  [2019-03-21 14:46:44,783] {jobs.py:1477} INFO - Starting the scheduler
+  # In a new shell, in Pipenv:
+```
+
+**Import DAGs to Local Airflow**
+
+This gets a local copy of the TUL COB DAGs into your local Airflow Environment for development purposes. By default, this local installation of Airflow expects DAGs to be places in `$AIRFLOW_HOME/dags`
+
+```
+(cob_datapipeline) $ mkdir $AIRFLOW_HOME/dags
+(cob_datapipeline) $ cd $AIRFLOW_HOME/dags
+(cob_datapipeline) $ git clone https://github.com/tulibraries/cob_datapipeline.git
+```
+There are quite a few ways you can get the cob_datapipeline DAGs into that directory; just ensure that the directory mirrors the order seen in the Github repository layout.
+
+**Set DAG Required Airflow Variables for Local Airflow**
+
+To load the required variables into your local airflow, you can perform the following process:
+
+```
+(cob_datapipeline) $ airflow variables -i airflow-variables.json
+  [2019-03-21 16:46:26,868] {__init__.py:51} INFO - Using executor SequentialExecutor
+  [2019-03-21 16:46:27,030] {models.py:160} WARNING - cryptography not found - values will not be stored encrypted.
+  13 of 13 variables successfully updated.
+```
+
+You should update that `airflow-variables.json` file to have the variables you need for your local development environment. Note: Airflow does then later update these variables during processes.
```

Comment:
```suggestion
You should update that `airflow-variables.json` file to have the variables you need for your local development environment, particularly the `AIRFLOW_DATA_DIR` `AIRFLOW_HOME` variables, replacing `~/airflow` with the absolute path to your airflow installation.
Note: Airflow does then later update these variables during processes.
```

Reply:
there's nothing particular about those 2, but minus the wording, sure.